content/spire/try/spire+aws-oidc-federation.md

```diff
@@ -0,0 +1,312 @@
+---
+title: AWS OIDC Authentication
+description: Using OIDC to Authenticate SPIRE to AWS S3
+weight: 3
+toc: true
+aliases:
+menu:
+  spire:
+    weight: 50
+    parent: 'spire-try'
+---
+
+This tutorial builds on the [Kubernetes Quickstart](/spire/try/getting-started-k8s/) guide to describe how a SPIRE identified workload can, using a JWT-SVID, authenticate to Amazon AWS APIs, assume an AWS IAM role, and retrieve data from an AWS S3 bucket. This avoids the need to create and deploy AWS IAM credentials with the workload itself.
+
+## Prerequisites
+
+Note the following required accounts, prerequisites, and limitations before starting on this tutorial:
+
+* Access to the Kubernetes environment that you configured when going through [Kubernetes Quickstart](/spire/try/getting-started-k8s/) 
+* This tutorial cannot run on Minikube as the Kubernetes environment must be network accessible by AWS
+* Access to the AWS console
+* [AWS CLI tools](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) installed on the same instance as the SPIRE Agent
+* DNS A record or CNAME for the SPIRE OIDC Discovery Provider REST API
+
+# Part One: Configure SPIRE Components
+
+In the first part of this procedure, you configure the SPIRE components. In the second part, you configure the AWS components. You test the connection in the third part.
+
+## Download Kubernetes YAML Files for this Tutorial
+
+Completing this tutorial requires a number of YAML files. To get these files, clone https://github.com/spiffe/spire-tutorials. The files for this tutorial are in the k8s/REPLACE_ME directory.
+
+## Replace Placeholder Strings in YAML Files
+
+The following strings in the YAML files must be substituted for values specific to your environment. Each location where you must make a change has been marked with `TODO:` in the YAML files.
+
+| String | Example Substitution | Description | Files to Change |
```

Comment:
Since users are starting with the k8s quickstart, can we just leave it as "demo-cluster"? In fact, maybe the following comments in the k8s quickstart should be removed:
\# NOTE: Change this to your cluster name
I ran through the k8s quickstart without changing any of the YAML files and it worked fine with "demo-cluster".